include/dlaf/matrix/tile.h

```diff
@@ -226,6 +239,12 @@ class Tile<const T, D> {
     return data_.ld() == data_.size().rows();
   }
 
+  /// Returns a subtile.
+  /// Note: to avoid segfaults or race conditions, the original tile must be kept in scope.
+  Tile subTileReference(const SubTileSpec& spec) const noexcept {
+    return Tile(*this, spec);
+  }
+
```

Comment:
I'm just trying to check my understanding of this:

This is _almost_ a copy constructor for `Tile` in the sense that the underlying memory chunk is correctly reference counted and kept alive, but the original tile is what keeps subsequent accesses from happening, and _that_ part is not reference counted?

Reply:
Summary of the separate discussion: This method is used to allow the usage of tile methods in tasks that execute more than one operation and do not work on full tiles.
The fact that the underlying memory is reference counted and kept alive is not important for this method (it is only an implementation detail and should not be relied on).